crawler/whitelist/thirdparty.yml

```
@@ -118,3 +118,7 @@
 - name: "PDFsam Basic"
   repos:
     - "https://github.com/torakiki/pdfsam"
+
+- name: "DXC-Industrialized-AI-Starter"
```

```suggestion
- name: "DXC Technology"
```

Since you have other repos (https://github.com/italia/developers-italia-backend/pull/272, https://github.com/italia/developers-italia-backend/pull/273), I'd put them all as a list in the `repos` key, using the `name` to describe the publisher.